fmt/src/git.rs

```diff
@@ -66,6 +63,6 @@ impl GitHelper {
         let path = path.canonicalize().context(ResolveAbsolutePathSnafu {
             path: path.display().to_string(),
         })?;
-        self.repo.is_path_ignored(path).context(GitOpSnafu)
+        todo!("how to get attributes and check ignore")
```

Comment:
Or using:

```rust
pub fn excludes(
        &self,
        index: &gix_index::State,
        overrides: Option<gix_ignore::Search>,
        source: gix_worktree::stack::state::ignore::Source,
    ) -> Result<AttributeStack<'_>, config::exclude_stack::Error> {
```

The last param is an enum, I can use the default.

The `overrides: Option<gix_ignore::Search>` I don't take a closer look, but hopefully `None` would work.

`index: &gix_index::State`? I don't have an idea.

Reply:
Seems I can use `let state = State::new(Kind::Sha1)` and hold it. Trying ...